src/daily-auths-report.tsx

```diff
@@ -207,7 +208,13 @@ function plot({
               thresholds: utcDay,
             },
             fill: agency ? "friendly_name" : "steelblue",
-            title: agency ? (bin: ProcessedResult[]) => bin[0]?.friendly_name : undefined,
+            title: (bin: ProcessedResult[]) => {
+              const date = yearMonthDayFormat(bin[0].date);
+              const total = formatWithCommas((bin || []).reduce((sum, d) => sum + d.count, 0));
```

Comment:
`bin` would never be falsey would it? Per argument type and the fact we access an index of it on the previous line.

```suggestion
              const total = formatWithCommas(bin.reduce((sum, d) => sum + d.count, 0));
```